_services/sp.textgridlab.org.yaml

```diff
@@ -5,4 +5,5 @@ website: 'https://sp.textgridlab.org/'
 dependencies:
   - '/infrastructure/gwdg-cloud'
   - '/servers/dfa'
+# ??????????????? Wird der hier benötigt im Status-Modell ???????????????????????
```

Comment:
i suggest leaving a `TODO` or `FIXME` marker here, but not a bunch of question marks.

Reply:
or may be even better: moving this [here](https://github.com/DARIAH-DE/status/issues/new/choose)

Reply:
done